src/families/stellar/js-getTransactionStatus.ts

```diff
@@ -124,6 +218,7 @@ const getTransactionStatus = async (
     warnings,
     estimatedFees,
     amount,
+    maxAmount,
```

Comment:
Sorry but you can't do this, `getTransactionStatus` is part of the `AccountBridge` contract and has a specified signature: https://github.com/LedgerHQ/ledger-live-common/blob/ef304028b3ce0c021b6c75f4a5ade38ccaec29c1/src/types/bridge.ts#L106

From what I can see you only use `maxAmount` in the `estimateMaxSpendable` function, but it already calls `getTransactionStatus` with a fake transaction that has `useAllAmount: true` : https://github.com/LedgerHQ/ledger-live-common/blob/ef304028b3ce0c021b6c75f4a5ade38ccaec29c1/src/families/stellar/js-estimateMaxSpendable.ts#L25
So why not just use the returned `amount` as done currently?